tests/test_venue_request.py

```diff
@@ -0,0 +1,139 @@
+        assert venue.review_stage_super_invitation
+        assert venue.meta_review_stage_super_invitation
+
+        assert venue.deploy_super_invitation
+        assert venue.comment_super_invitation
+        assert venue.recruitment_super_invitation
+
+    def test_venue_request_post_deploy_revise(self, client, selenium, request_page, helpers):
+        
+        super_id = 'openreview.net'
+        support_group_id = super_id + '/Support'
+        VenueRequest(client, support_group_id, super_id)
+        
+        time.sleep(5)
+        request_page(selenium, 'http://localhost:3000/group?id={}&mode=default'.format(support_group_id), client.token)
+
+        helpers.create_user('new_test_user@mail.com', 'Newtest', 'User')
+        request_form_note = client.post_note(openreview.Note(
+            invitation=support_group_id +'/-/Request_Form',
+            signatures=['~Newtest_User1'],
+            readers=[
+                support_group_id,
+                '~Newtest_User1',
+                'new_test_user@mail.com',
+                'tom@mail.com'
+            ],
+            writers=[],
+            content={
+                'title': 'Test 2021 Venue',
+                'Official Venue Name': 'Test 2021 Venue',
+                'Abbreviated Venue Name': 'TestVenue@OR2021',
+                'Official Website URL': 'https://testvenue2021.gitlab.io/venue/',
+                'program_chair_emails': [
+                    'new_test_user@mail.com',
+                    'tom@mail.com'],
+                'contact_email': 'new_test_user@mail.com',
+                'Area Chairs (Metareviewers)': 'No, our venue does not have Area Chairs',
+                'Venue Start Date': '2020/02/01',
+                'Submission Deadline': '2025/09/30',
+                'Location': 'Virtual',
+                'Paper Matching': [
+                    'Reviewer Bid Scores',
+                    'Reviewer Recommendation Scores'],
+                'Author and Reviewer Anonymity': 'Single-blind (Reviewers are anonymous)',
+                'Open Reviewing Policy': 'Submissions and reviews should both be private.',
+                'Public Commentary': 'Yes, allow members of the public to comment non-anonymously.',
+                'How did you hear about us?': 'ML conferences',
+                'Expected Submissions': '100'
+            }))
+        
+        assert request_form_note
+        request_page(selenium, 'http://localhost:3000/forum?id=' + request_form_note.forum, client.token)
+
+        messages = client.get_messages(
+            to='new_test_user@mail.com',
+            subject='Your request for OpenReview service has been received.')
+        assert messages and len(messages) == 1
+        assert messages[0]['content']['text'] == 'You recently requested conference management services from OpenReview. A member of our support team will contact you shortly. You can view the request here: https://openreview.net/forum?id=' + request_form_note.forum
```

Comment:
I want to change the wording of this message. I would change 'conference management services' -> 'host of venue' or similar what we show in the request venue home page.

'A member of our support team will contact you shortly' -> 'We will review your request and comment will be posted on your request when the venue is deployed'.

I would mention the FAQ here too.

@zbialecki suggestions?

Reply:
I would say, 
"Thank you for choosing OpenReview to host your upcoming venue. We are reviewing your request and will post a comment on the request forum when the venue is deployed. You can access the request forum here: https://openreview.net/forum?id=' + request_form_note.forum"

Reply:
Done.